dimensional.cabal

```diff
@@ -62,7 +63,8 @@ library
                        Numeric.Units.Dimensional.UnitNames,
                        Numeric.Units.Dimensional.UnitNames.InterchangeNames,
                        Numeric.Units.Dimensional.Variants
-  other-modules:       Numeric.Units.Dimensional.UnitNames.Internal
+  other-modules:       Numeric.Units.Dimensional.Internal,
```

Comment:
Is this really exposed to library users, or only "internally" in the package? My impression was the latter. I.e., that outside of dimensional package an `import Numeric.Units.Dimensional.Internal` will fail. Is that correct, and is that what you intended?